packages/backend-tasks/src/tasks/PluginTaskSchedulerImpl.ts

```diff
@@ -103,6 +105,16 @@ export class PluginTaskSchedulerImpl implements PluginTaskScheduler {
       },
     };
   }
+
+  async getTaskIds(): Promise<TaskIds> {
+    const knex = await this.databaseFactory();
+    const taskStore = new TaskStore(knex);
```

Comment:
Oh man, just put a method here instead :) A wrapping single-method single-line class feels a bit too close to Java for comfort.